docs/docs/ci.md

```diff
@@ -20,6 +20,46 @@ While each repository is free to define whatever scripts it requires, most RAPID
 Some repositories will include helper scripts in this directory to share logic between the above scripts.
 This practice is especially common if there are multiple wheels to build since the logic for building various wheels is usually similar.
 
+### Wheel-building scripts
+
+Some RAPIDS wheels rely on other wheels to provide build-time dependencies.
+For example, `pylibcudf-cu12` wheel builds install `libcudf-cu12` wheels to get `libcudf.so` and its headers.
+
+By default, `pip wheel` or `python -m build` will install such build-time dependencies into a temporary directory (see ["Python Packages"](build.md#python-packages) for details).
+The path to that directory is randomly chosen, and so the path that those build dependencies are installed to changes on every build.
```

Comment:
As above, I would be comfortable rephrasing this assuming the reader knows what build isolation is and simply providing the informational link for them if they need it.

Reply:
I don't think that "knows what build isolation is" == "knows that the path is randomly chosen every time, and how that affects `sccache`".

In my opinion, this problem is sufficiently subtle, and the explanation sufficiently brief, that this should be kept in.

Reply:
Fair enough, that's a good argument for keeping the detail.